modules/openapi-generator/src/main/resources/JavaSpring/converter.mustache

```diff
@@ -0,0 +1,34 @@
+package org.openapitools.configuration;
```

Comment:
probably this has to be smth like `package {{package}}.configuration`;

Reply:
I change for {{configPackage}}